x-pack/test_serverless/functional/test_suites/search/inference_management.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ModelConfig } from '@kbn/inference_integration_flyout/types';
+
+import { testHasEmbeddedConsole } from './embedded_console';
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const pageObjects = getPageObjects([
+    'svlCommonPage',
+    'svlCommonNavigation',
+    'inferenceManagement',
+  ]);
+  const browser = getService('browser');
+  const ml = getService('ml');
+
+  describe('Serverless Inference Management UI', function () {
+    this.tags('skipMKI');
```

Comment:
if we're enabling the feature flag then we want to remove this tag. Enabling the feature means we WANT to run these against MKI.

We should verify the tests against MKI before merging this.